src/views/wedo2/wedo2.jsx

```diff
@@ -44,6 +44,14 @@ var Wedo2 = React.createClass({
                                 </h4>
                                 <p>
                                     <FormattedHTMLMessage id='wedo2.installText' />
+                                    <br />
+                                    <a href="https://itunes.apple.com/WebObjects/MZStore.woa/wa/viewSoftware?id=1084869222&mt=12">
+                                        <FormattedMessage id='wedo2.downloadMac' />
+                                    </a>
+                                    <br />
+                                    <a href="/">
```

Comment:
Should this be a link to the Device Manager setup on downloads.scratch.mit.edu? e.g:
http://downloads.scratch.mit.edu/device-manager/Scratch%20Device%20Manager%20Setup%201.1.0.exe